index.html

```diff
@@ -57,93 +57,95 @@
+						</div>
+						<div class="crow">
+							<label for="gasLimit"><input type="number" id="gasLimit" value="3000000"> Transaction gas limit</label>
+						</div>
+						<div class="crow hide">
+							<label for="gasPrice"><input type="number" id="gasPrice" value="0"> Gas Price</label>
+						</div>
+						<div class="crow">
+							<label for="value"><input type="text" id="value" value="0"> Value (e.g. .7 ether or 5 wei, defaults to ether)</label>
+						</div>
 						<span id="executionContext">
-							<label for="vm-mode">
-								<input id="vm-mode" type="radio" value="vm" checked name="executionContext">
-								<strong>JavaScript VM</strong>
-								<p>Execution environment does not connect to any node, everything is local and in memory only.</p>
-							</label>
-							<label for="injected-mode">
-								<input id="injected-mode" type="radio" value="injected" checked name="executionContext">
-								<strong>Injected Web3</strong>
-								<p>Execution environment has been provided by Mist or similar provider.</p>
-							</label>
-							<label for="web3-mode">
-								<input id="web3-mode" type="radio" value="web3" name="executionContext">
-								<strong>Web3 Provider</strong>
-																<p>Execution environment connects to node at localhost (or via IPC if available), transactions will be sent to the network and can cause loss of money or worse!<br/>
-																<b>If this page is served via https and you access your node via http, it might not work. In this case, try cloning the repository and serving it via http.</b></p>
-								<label for="web3Endpoint">
-									<strong>Web3 Provider Endpoint</strong>: <input type="text" id="web3Endpoint" value="http://localhost:8545">
-								</label>
-							</label>
+							Select execution environment: <br><brs>
+							<select id='selectExEnv'>
+							  <option
+							    id="vm-mode"
+							    title="Execution environment does not connect to any node, everything is local and in memory only."
+							    value="vm"
+							    checked name="executionContext">
+							    JavaScript VM
+							  </option>
+							  <option
+							    id="injected-mode"
+							    title="Execution environment has been provided by Mist or similar provider."
+							    value="injected"
+							    checked name="executionContext">
+							    Injected Web3
+							  </option>
+						    <option
+						      id="web3-mode"
+						      title="Execution environment connects to node at localhost (or via IPC if available), transactions will be sent to the network and can cause loss of money or worse!
+						      If this page is served via https and you access your node via http, it might not work. In this case, try cloning the repository and serving it via http."
+						      value="web3"
+						      name="executionContext">
+						      Web3 Provider
+						    </option>
+							</select>
```

Comment:
@axic I brushed a bit my Index.html, looks good in my editor, hope all is fine now.
d09f8ad Fix styling issues in Index.html is the commit 👍